index.html

```diff
@@ -27,94 +33,102 @@ <h4 class="connect">LET'S CONNECT</h4>
         </ul>
     </article>
     <section class="grid-container">
-
-        <section class="works">
-            <a class="snapshoots"><img src="./assets/Snapshoot1.png" width="295" height="220" alt="snapshoot1"></a>
-            <h2 class="work-title">Tonic</h2>
-            <section class="sub-desc">
-                <h5 class="brand">CANOPY</h5>
-                <img src="./assets/Counter.png" alt="dot">
-                <p class="stack">Back End Dev</p>
-                <img src="./assets/Counter.png" alt="dot">
-                <p class="year">2015</p>
-            </section>
-            <p class="description">
-                A daily selection of privately personalized reads; no accounts or sign-ups required.
-            </p>
-            <ul class="languages">
-                <li class="lang">html</li>
-                <li class="lang">css</li>
-                <li class="lang">javascript</li>
-            </ul>
-            <button type="button" class="detail-btn">See Project</button>
+        <section class="work left">
+            <a class="snapshoots"><img class="big-img" src="./assets/Snapshoot1.png" alt="Snapshoot1"></a>
+            <div class="info">
+                <h2 class="work-title">Tonic</h2>
+                <div class="sub-desc">
+                    <h5 class="brand">CANOPY</h5>
+                    <img src="./assets/Counter.png" alt="dot">
+                    <p class="stack">Back End Dev</p>
+                    <img src="./assets/Counter.png" alt="dot">
+                    <p class="year">2015</p>
+                </div>
+                <p class="description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="languages">
+                    <li class="lang">html</li>
+                    <li class="lang">css</li>
+                    <li class="lang">javascript</li>
+                </ul>
+                <button type="button" class="detail-btn">See Project</button>    
+            </div>
```

Comment:
- [ ] For all work cards please add the right width value and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/159918572-9c95212b-8926-4a79-a852-345653769bb3.png) |  ![image](https://user-images.githubusercontent.com/68796232/159918596-1e61541c-5ed7-4d58-8f8d-4f99c8f753ad.png) |